contracts/posts/Feed.sol

```diff
@@ -35,44 +31,36 @@ contract Feed is Operated, EventMetadata, Template {
 
     // state functions
 
-    function createPost(address postFactory, bytes memory callData) public returns (address post) {
+    function submitProofHash(bytes memory proofHash) public {
```

Comment:
I've been thinking that there may be utility in adding a `proofHash` parameter to the Feed's `initialize` call, so that when a Feed is created, it has a single post by default.

```
function initialize(
        address operator,
        bytes memory metadata,
        bytes memory proofHash
    ) 
```

Or maybe the solution to this is to have a proxy contract that wraps the `initialize` and `submitProofHash` in a single transaction